lang/LangSource/PyrLexer.cpp

```diff
@@ -2099,7 +2099,6 @@ void shutdownLibrary()
 	gLangMutex.unlock();
 	deinitPrimitives();
 
-	SC_LanguageConfig::freeLibraryConfig();
```

Comment:
On shutdownLibrary() library is not deleted in order to be able to use in-memory version on recompile.